pkg/network/setup/netstat.go

```diff
@@ -119,13 +124,37 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 
 	interfacesStatus = ifacesStatusFromMultus(interfacesStatus, multusStatusNetworksByName, vmiInterfacesSpecByName)
 
+	interfacesStatus = append(interfacesStatus, reportMissingNetworks(vmi.Spec.Networks, interfacesStatus)...)
+
+	reportPodIfaceNames(interfacesStatus, podIfacesNamesByNetworkNames)
```

Comment:
I think the pattern followed in this function is to return the data and not to show it is mutated.
Even if you do mutate it, I think you should return the slice back.

Reply:
This code was removed.